CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # GCP Tamr IAM module
 
+## v1.1.1 - April 5th 2023
+* Force create service account
```

Comment:
Make sure to be specific when there are changes to the variables. It's very helpful when you need to upgrade a module to read through the changelog.

For example:
https://github.com/Datatamer/terraform-aws-emr/blob/master/CHANGELOG.md#v900---august-23rd-2022